apps/block_scout_web/lib/block_scout_web/controllers/api/rpc/address_controller.ex

```diff
@@ -508,6 +525,34 @@ defmodule BlockScoutWeb.API.RPC.AddressController do
     end
   end
 
+  defp list_pending_withdrawals(address_hash) do
+    case Chain.pending_withdrawals_for_account(address_hash) do
+      [] ->
+        {:error, :not_found}
+
+      pending_withdrawals ->
+        {:ok,
+         [
+           %{
+             total:
+               Enum.reduce(pending_withdrawals, Decimal.new(0), fn withdrawal, acc ->
+                 Decimal.add(Wei.to(withdrawal.amount, :wei), acc)
+               end),
+             available_for_withdrawal:
+               Enum.reduce(pending_withdrawals, Decimal.new(0), fn withdrawal, acc ->
+                 if DateTime.diff(withdrawal.available, DateTime.utc_now()) <= 0 do
```

Comment:
What if we move this logic to the `CeloUnlocked` module? It would be a little more readable and all related logic would be localized, like
```if CeloUnlocked.is_available?(withdrawal)```